### PR TITLE
LPD-34076 Fixing notification template variables in the 'general-variables', 'Util' and 'Display Page' category don't work

### PR DESCRIPTION
**Pull request by victorKammerer**
Jira ticket related: [LPD-34076](https://liferay.atlassian.net/browse/LPD-34076)

[LPD-34076]: https://liferay.atlassian.net/browse/LPD-34076?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by victorKammerer** [comment]
ci:test:sf

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java [hunk 1]
```diff
@@ -584,6 +584,15 @@ private String _formatBody(
 			if (httpServletRequest != null) {
 				template.put(
 					"portalURL", portal.getPortalURL(httpServletRequest));
+				template.put(
```
Comment (linolaoj) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java:587]: Hi @victorKammerer, can we reorder here?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by victorKammerer** [comment]
ci:test:sf

**Review by linolaoj — APPROVED** [review]

**Comment by linolaoj** [comment]
ci:test:objects

**Comment by linolaoj** [comment]
ci:test:object

**Comment by linolaoj** [comment]
ci:forward

**Comment by linolaoj** [comment]
ci:stop

**Comment by linolaoj** [comment]
Hi @victorKammerer , the error of `testFreeMarkerNotification` is related.

**Comment by victorKammerer** [comment]
ci:test:relevant

File: modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateFTLElementsMVCResourceCommand.java [hunk 2]
```diff
@@ -140,11 +141,30 @@ private void _fillTemplateContextTemplateVariables(
 		TemplateVariableGroup generalVariablesTemplateVariableGroup =
 			templateVariableGroupsMap.get("general-variables");
 
+		TemplateVariableGroup copyGeneralVariablesTemplateVariableGroup =
```
Comment (linolaoj) [inline modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateFTLElementsMVCResourceCommand.java:144]: Hi @victorKammerer , I checked in the portal with `git grep "new TemplateVariableGroup"` and we always initialize this inside of a class `TemplateHander` on this class we can also set the restricted variable.

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by victorKammerer** [comment]
ci:test:sf

File: modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateFTLElementsMVCResourceCommand.java (hunk 2 above)
Comment (victorKammerer) [reply modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateFTLElementsMVCResourceCommand.java]: Done, @linolaoj! I created and implemented a new `NotificationTemplatesTemplateHandler` class

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:forward

**Comment by linolaoj** [comment]
Hi @victorKammerer, can you please check the test from [notification-web/emailNotificationTemplate.spec.ts](https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/425318/testReport/notification-web_emailNotificationTemplate.spec/ts), it might be related.

**Comment by linolaoj** [comment]
ci:forward

**Comment by victorKammerer** [comment]
ci:forward

**Comment by gabrielwas** [comment]
@victorKammerer Did you run all automated tests related to notifications on your local machine before runnig CI?

cc @linolaoj

**Comment by victorKammerer** [comment]
> @victorKammerer Did you run all automated tests related to notifications on your local machine before running CI?
> 
> cc @linolaoj

I ran the ones at the `EmailNotificationTypeTest`.

**Pull request closed without merging**